src/AppBundle/Repository/DoctrineORMClmRaidRepository.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace AppBundle\Repository;
+
+use AppBundle\Entity\ClmRaid;
+use Doctrine\ORM\EntityRepository;
+use Doctrine\ORM\NoResultException;
+use Symfony\Component\Config\Definition\Exception\Exception;
+
+class DoctrineORMClmRaidRepository extends EntityRepository implements ClmRaidRepositoryInterface
+{
+    /**
+     * @param ClmRaid $raid
+     */
+    public function save(ClmRaid $raid)
+    {
+        $em = $this->getEntityManager();
+
+        $em->persist($raid);
+        $em->flush();
+    }
+
+    /**
+     * @param int $id
+     * @return ClmRaid
+     */
+    public function findById($id)
+    {
+        $query = $this->createQueryBuilder('b')
+            ->where('b.id = :parId')
```

Comment:
Both terms very common in pure sql syntax:
b is that short on purpose and used as an alias for e.g. a table to reference that table more easily. Especially useful for long join querys referencing several tables at once.
`select * from user as u where u.name =hans`
Also named query parameters are usually prefixed by "par".
So :parId is very descriptive.
'b' could be named 'raid_table' for convenience or 'r'